git/mocks/mockPackageJson.js

```diff
@@ -0,0 +1,24 @@
+var packageWithVersion = {
+  "name": "dgeni-packages",
+  "version": "0.10.13",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/owner/repo.git"
+  }
+}
+
+var packageWithBranchVersion = {
+  "name": "dgeni-packages",
+  "version": "0.10.13",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/owner/repo.git"
+  }
+}
```

Comment:
Yeah, I am still planning on adding some addtional tests to test the `angular.js` code branches that handle `branchPattern` and `branchVersion`